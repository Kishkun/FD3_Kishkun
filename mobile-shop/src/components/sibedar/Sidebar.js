import React from "react";
import BasketCart from "../basketCart/BasketCart";
import Search from "../search/Search";
import Categories from "../categories/Categories";

const Sidebar = () => {

    return (
        <div>
            <BasketCart/>
            <Search/>
            <Categories/>
        </div>
    )
};

export default Sidebar;