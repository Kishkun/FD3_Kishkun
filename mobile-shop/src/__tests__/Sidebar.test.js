import React from "react";
import {shallow} from 'enzyme';
import Sidebar from "../components/sibedar/Sidebar";

test('Sidebar test component', () => {

    const wrapper = shallow (
        <Sidebar />
    );

    expect(wrapper).toMatchSnapshot();
});