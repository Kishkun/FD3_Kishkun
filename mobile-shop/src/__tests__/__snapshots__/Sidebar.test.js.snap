// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar test component 1`] = `
<div>
  <Connect(BasketCart) />
  <Connect(Search) />
  <withRouter(Connect(Categories)) />
</div>
`;
